utils/redis_helper.py

```diff
@@ -54,4 +54,48 @@ def push_object(cls, key, obj, queryset):
         serialized_data = DjangoModelSerializer.serialize(obj)
         conn.lpush(key, serialized_data)
         # to ensure after push the new object, we stay within the length
-        conn.ltrim(key, 0, settings.REDIS_LIST_LENGTH_LIMIT - 1)
\ No newline at end of file
+        conn.ltrim(key, 0, settings.REDIS_LIST_LENGTH_LIMIT - 1)
+
+    @classmethod
+    def get_count_key(cls, obj, attr):
+        # eg, tweet.likes_count:id
+        return '{}.{}:{}'.format(obj.__class__.__name__, attr, obj.id)
+
+    @classmethod
+    def incr_count(cls, obj, attr):
```

Comment:
This is how the resolve the issue that invalidate the data in cache too frequently